lib/view/subject_card.dart

```diff
@@ -53,22 +57,63 @@ class _SubjectCardState extends State<SubjectCard> {
                                   ))
                             ]),
                         Row(
-                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
                           children: <Widget>[
                             Column(
                               mainAxisSize: MainAxisSize.min,
-                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              crossAxisAlignment: CrossAxisAlignment.start,
                               children: <Widget>[
-                                Text(widget.subject.type.toString()),
-                                Text(widget.subject.dueDate.toIso8601String()),
-                                Text(widget.subject.room.toString()),
-                                Text(widget.subject.description)
+                                Text(
+                                  widget.subject.type.toString(),
+                                  style: TextStyle(
+                                    color: Colors.black87,
+                                    fontSize: 17,
+                                  ),
+                                ),
+                                Text(
+                                  formatDate(widget.subject.dueDate,
+                                      [dd, '/', mm, '/', yyyy]).toString(),
```

Comment:
just my optinion: maybe we can add the Time too, because i think it is a needed information.